client/package.json

```diff
@@ -37,6 +38,7 @@
         "enzyme": "^3.11.0",
         "enzyme-adapter-react-16": "^1.15.3",
         "jss-rtl": "^0.3.0",
+        "lodash": "^4.17.20",
```

Comment:
lodash is pretty heavy and youre only using ._get... 
i would consider adding this function since its pretty short

Reply:
could refer to https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_get